docs/builtins/Time/parse-time.md

```diff
@@ -2,59 +2,8 @@
-| `%Q` | The fraction of a second, up to six second decimal places, without trailing zeros. For a whole number of seconds, `%Q` produces an empty string.|
-| `%r` | The complete calendar time including the AM/PM format as specified for the current locale, for example, `%I:%M:%S %p`.|
-| `%R` | The hour and minute in decimal numbers using the format `%H:%M`.|
-| `%s` | The number of whole seconds since the UNIX epoch (since 1970-01-01 00:00:00 UTC). For times before the UNIX epoch, this is a negative number. Note that in `%s.%q` and `%s%Q` formats, the decimals are positive, not negative. For example, 0.9 seconds before the UNIX epoch is formatted as `"-1.1"` with `%s%Q`.|
-| `%S` | The seconds of a minute, zero-padded to two characters, for example, `"00"`–`"60"`.|
-| `%T` | The time of day using the format `%H:%M:%S`.|
-| `%u` | The day of the week as a number with Monday being 1, for example, `"1"`–`"7"`.|
-| `%U` | The week number of the year as a number, starting with the first Sunday as the first day of the first week, zero-padded to two characters, for example, `"00"`–`"53"`.|
-| `%v` | The microsecond of a second, zero-padded to six characters, for example, `"000000"`–`"999999"`.|
-| `%V` | The week number as specified using the ISO 8601:1988 standard where weeks start with Monday and end with Sunday. zero-padded to two characters, for example,`"01"`–`"53"`.|
-| `%w` | The day of the week as a number, starting with Sunday as 0, for example,`"0"` (= Sunday) – `"6"` (= Saturday).|
-| `%W` | The week number of the year as a number (range 00 through 53), starting with the first Monday as the first day of the first week. zero-padded to two characters, for example, `"00"`–`"53"`.|
-| `%x` | The preferred date representation as specified for the current `locale`, for example, `%m\/%d\/%y`.|
-| `%X` | The preferred time of day representation as specified for the current locale, for example, `%H:%M:%S`.|
-| `%y` | The year without a century as a number, zero-padded to two characters, for example, `"00"`–`"99"`.|
-| `%Y` | The full year as a number with no padding.|
-| `%z` | Numeric representation of the time zone using the RFC 822/ISO 8601:1988 standard, for example, `"-0600"` or `"+0100"`.|
-| `%Z` | The time zone abbreviation or empty if the time zone can't be determined.|
-
-Note that two percentage characters (`%%`) are interpreted as a literal percentage sign (`%`), not a time formatting template.
-In addition, the template for picoseconds with zero padding (`%q`) doesn't work properly, so it's not included as a supported template for formatting time strings.
-
-<!-- NEXT GEN DOCS
-The following table provides a summary of common time formatting codes:
+The `parse-time` function accepts format codes that are derived from the formatting time `strftime` function time templates.
+The following table provides a summary of the most common time formatting codes:
```

Comment:
See: other comment, and remove "the following table"

Reply:
Same.